git-release.ps1

```diff
@@ -72,14 +88,15 @@ if ($dryRun) {
     if ($config.remote -ne $nil) {
         $gitDeletions = $toRemove | ForEach-Object { ":$_" }
 
-		if ($config.atomicPushEnabled) {
-			git push --atomic $config.remote "$($config.remote)/$($branchName):$target" @gitDeletions "$($commitish):refs/heads/$($config.upstreamBranch)"
-		} else {
-			git push $config.remote "$($config.remote)/$($branchName):$target" @gitDeletions "$($commitish):refs/heads/$($config.upstreamBranch)"
-		}
+        $atomicPart = $config.atomicPushEnabled ? @("--atomic") : @()
+        $releasePart = $cleanupOnly ? @() : @("$($config.remote)/$($branchName):$target")
+
+        git push @atomicPart $config.remote @releasePart @gitDeletions "$($commitish):refs/heads/$($config.upstreamBranch)"
```

Comment:
@Mike343, I rewrote your logic in this part. Please take a look and make sure it still works for you!